ogr/services/github.py

```diff
@@ -103,31 +104,55 @@ def __init__(
         read_only=False,
         github_app_id: str = None,
         github_app_private_key: str = None,
+        github_app_private_key_path: str = None,
         **_,
     ):
         super().__init__()
         self.token = token
 
         # Authentication via GitHub app
         self.github_app_id = github_app_id
-        self.github_app_private_key = github_app_private_key
+        self._github_app_private_key = github_app_private_key
+        self.github_app_private_key_path = github_app_private_key_path
 
         self.github = github.Github(login_or_token=self.token)
         self.read_only = read_only
 
+    @property
+    def github_app_private_key(self):
+        if self._github_app_private_key:
+            return self._github_app_private_key
+
+        if self.github_app_private_key_path:
+            if not Path(self.github_app_private_key_path).is_file():
+                raise GithubAPIException(
+                    f"Path to the github-app private key "
+                    f"({self.github_app_private_key_path}) "
+                    f"does not exist."
+                )
+            return Path(self.config.github_app_cert_path).read_text()
```

Comment:
copy-pasta error? I'd expect
```python
            self._github_app_private_key = Path(self.github_app_private_key_path).read_text()
            return self._github_app_private_key
            
```

Reply:
Thanks!

Fixed.

Reply:
me not satisfied :-)

What's the `self._github_app_private_key` good for when do you don't set it anywhere?